__init__.py

```diff
@@ -0,0 +1 @@
+from zeppelin_converter import ZeppelinConverter
```

Comment:
I almost forgot, you should move this file to the zeppelin package. And maybe remove this import 😄